Packs/TaniumThreatResponse/Integrations/TaniumThreatResponseV2/TaniumThreatResponseV2.yml

```diff
@@ -4,2191 +4,2351 @@ commonfields:
   id: Tanium Threat Response v2
   version: -1
 configuration:
-- display: Hostname, IP address, or server URL
```

Comment:
lets avoid the big diff here, and apply here only the relevant changes

Reply:
done, its a consequence of the plugin, so i just did it manually instead